lib/linguist/heuristics.yml

```diff
@@ -447,6 +455,10 @@ disambiguations:
     pattern: '^%(end|ctor|hook|group)\b'
   - language: Linker Script
     pattern: 'OUTPUT_ARCH\(|OUTPUT_FORMAT\(|SECTIONS'
+- extensions: ['.xml']
+  rules:
+  - language: ObjectScript
+    pattern: '<Export generator="(Cache|IRIS)" version="\d+"'
```

Comment:
As mentioned elsewhere, we have an XML strategy so all XML files will be identified as such.

Please remove this.

Reply:
It works exactly as expected, and tests profs it.

Reply:
See my comment above https://github.com/github/linguist/pull/4480#discussion_r271683745